contrib/parseq-restli-client/src/main/antlr4/com/linkedin/restli/client/config/RequestConfigKey.g4

```diff
@@ -3,7 +3,7 @@ grammar RequestConfigKey;
 key 			: inbound '/' outbound EOF;
 inbound			: ( restResource | '*' ) '.' ( operationIn | '*' );
 outbound		: ( restResource | '*' ) '.' ( operationOut | '*' );
-restResource 	: Name ( ':' Name )*;
+restResource 	: Name ( '-' Name )* ( ':' Name ( '-' Name )* )*;
```

Comment:
I think colo can be only specified on resource name, not sub-resource, so it should be:
restResource 	: Name ( '-' Name )* ( ':' Name )*;

Reply:
Yes, you are right. sub-resource should not have multi-colo support, fixed.